components/tabs/index.css

```diff
@@ -177,6 +185,10 @@
 	}
 }
 
+.spectrum-Tabs--vertical {
+	--mod-tabs-list-background-direction: var(--mod-tabs-list-background-direction-vertical, right);
```

Comment:
This (and the other `background-direction` variables below) were not working prefixed with `--spectrum`, they've been changed to `--mod` which matches what is seen in `main`, see https://github.com/adobe/spectrum-css/blob/main/components/tabs/index.css#L112